modules/component/src/core/render-creator.ts

```diff
@@ -1,13 +1,11 @@
 import { ChangeDetectorRef, NgZone } from '@angular/core';
 
-import { hasZone } from '../utils/has-zone';
-
 export interface RenderConfig {
   ngZone: NgZone;
   cdRef: ChangeDetectorRef;
 }
 
-export function createRender<T>(config: RenderConfig): () => void {
+export function createRender(config: RenderConfig): () => void {
```

Comment:
This was existing but seems like this function should have a different name like `createRenderDetection` or something similar that's more accurate of what its doing.